.gitignore

```diff
@@ -1 +1,2 @@
 /vendor/
+composer.lock
```

Comment:
Why exclude composer.lock, don't we need it to pin down dependencies?

Reply:
`composer.lock` is only used for the root repo. The lock file for dependencies are ignored and it doesn't make sense to keep it for projects that will only ever be used as dependencies.